### PR TITLE
Restructure directories, fix createcalaccessrawmodeldocs command

### PR DESCRIPTION
**Pull request by bcipolli**
Per #42, I restructured the directories (particularly `disclosure_backend`). This includes:
- Move `disclosure_backend/project/*` into `disclosure_backend`.
- Move  `disclosure_backend/toolbox/templates/` to `disclosure_backend/templates`; remove `toolbox` (otherwise empty).
- Make a `disclosure_backend/tests/` directory, and move test-related info there.

In trying to verify that everything works after the restructure, I found a bug in `createcalaccessrawmodeldocs` (#95), then fixed it to make sure this restructuring is good. I also added a basic regression test for `createcalaccessrawmodeldocs`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ We're going to create an environment with Python 2.7.9 for the project
   mysql> \q
   ```
 
-2. Create `disclosure-backend/project/settings_local.py`
+2. Create `disclosure-backend/settings_local.py`
```
Comment (tdooner) [inline README.md]: nit: `disclosure_backend`

**Comment by tdooner** [comment]
Great. Thanks for reorganizing this project to be easier to understand. I'm getting the feeling that you're going to teach me a lot of neat python tricks. :+1:

File: README.md (hunk 1 above)
Comment (bcipolli) [inline README.md]: Nice find, eagle eye :eyes: :smile: Fixed and I think this is ready to go!

**Comment by adborden** [comment]
Looks great, @bcipolli do you need help merging this?

**Comment by bcipolli** [comment]
Thanks! Nope, just need to rebase from the change you just merged. Will do later tonight!

**Comment by bcipolli** [comment]
Check that--easy rebase, no conflicts. @adborden If you're good, please merge! heading home before tonight's meeting.

**Pull request merged**